tests/onnx/data/reference_graphs/quantization/synthetic/float64_model.dot

```diff
@@ -0,0 +1,15 @@
+strict digraph  {
+"0 Reciprocal" [id=0, type=Reciprocal];
+"1 Cast" [id=1, type=Cast];
+"2 QuantizeLinear_Cast_Y_1" [id=2, type=QuantizeLinear];
+"3 DequantizeLinear_Cast_Y_1" [id=3, type=DequantizeLinear];
+"4 Mul" [id=4, type=Mul];
+"5 nncf_model_input_0" [id=5, type=nncf_model_input];
+"6 nncf_model_output_0" [id=6, type=nncf_model_output];
+"0 Reciprocal" -> "1 Cast"  [label="[1, 3, 10, 10]", style=dashed];
+"1 Cast" -> "2 QuantizeLinear_Cast_Y_1"  [label="[1, 3, 10, 10]", style=solid];
+"2 QuantizeLinear_Cast_Y_1" -> "3 DequantizeLinear_Cast_Y_1"  [label="[1, 3, 10, 10]", style=dashed];
+"3 DequantizeLinear_Cast_Y_1" -> "4 Mul"  [label="[1, 3, 10, 10]", style=solid];
```

Comment:
Why do the MatMul weights are not quantized?

Reply:
It is not MatMul, it is elementwise Mul operation